flink-end-to-end-tests/test-scripts/test_quickstarts.sh

```diff
@@ -18,29 +18,38 @@
 ################################################################################
 
 # End to end test for quick starts test.
+# Usage:
+# FLINK_DIR=<flink dir> flink-end-to-end-tests/test-scripts/test_quickstarts.sh <TestFileName> <Type (java or scala)>
```

Comment:
not sure how much value the `TestFileName` adds, given that it's the same for both tests and the package being hard-coded. At the very least it should be called `TestClassName`.

Reply:
will change the name